docs/user/Build.md

```diff
@@ -1,5 +1,8 @@
 # Build OpenROAD Flow
 
+We support three major ways of installation. They are: [Docker](./BuildWithDocker.md), [Prebuilt Binaries](./BuildWithPrebuilt.md), and finally [local installation](./BuildLocally.md). You may also choose to follow this script to customise your build process. 
```

Comment:
We support following three major ways of installation. 
- [Docker](./BuildWithDocker.md)
- [Prebuilt Binaries](./BuildWithPrebuilt.md)
- [Local Installation](./BuildLocally.md)

You may also choose to follow this script to customise your build process.